.goreleaser.yaml

```
@@ -0,0 +1,37 @@
+# Copyright 2023 Adevinta
+
+version: 1
+
+project_name: lava
+
+gomod:
+  proxy: true
+
+builds:
+  - id: lava
+    main: ./cmd/lava
+    env:
+      - CGO_ENABLED=0
+    goos:
+      - linux
+      - darwin
+    goarch:
+      - 386
+      - amd64
+      - arm
+      - arm64
+    ignore:
+      - goos: darwin
+        goarch: arm
+    mod_timestamp: '{{.CommitTimestamp}}'
+    ldflags:
+      - -s -w
+
+archives:
+  - format: tar.gz
```

```suggestion
  - format: binary
```

I think binaries are more convenient than archives. It removes the need of extracting them before run. Can we test it in [seilagamo/poc-lava-release#1](https://github.com/seilagamo/poc-lava-release/pull/1)?

More info: https://goreleaser.com/customization/archive/#archives

---

Discarded due to the different in size of the generated binaries.